apps/src/StudioApp.js

```diff
@@ -608,6 +608,61 @@ StudioApp.prototype.init = function (config) {
   }
 };
 
+StudioApp.prototype.hasValidContainedLevelResult_ = function () {
+  var results = this.getContainedLevelResults();
+  if (results.length !== 1) {
+    throw "Exactly one contained level result is currently required.";
+  }
+  var firstResult = results[0];
+  return typeof firstResult.result.response !== 'undefined' &&
+      firstResult.result.response !== '';
+};
+
+StudioApp.prototype.notifyInitialRenderComplete = function (config) {
```

Comment:
Is this parameter the full level config object?  Something I've been trying to get us to do (with little success, admittedly) is annotate our `config` parameters using [this `AppOptionsConfig` typedef](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/StudioApp.js#L69) so it's obvious where we're passing the whole level config blob around.

```javascript
/**
 * @param {!AppOptionsConfig} config
 */
StudioApp.prototype.notifyInitialRenderComplete = function {
```